README.md

```diff
@@ -107,6 +107,42 @@ RSpec.describe "Posts Controller", type: :request do
       response 200, description: "success"
     end
   end
+
+  # Path Update Object
+  path '/posts/{post_id}' do
```

Comment:
I like having a POST operation that demonstrates how the body should work. Could we fold this into the `path '/posts/{post_id}' do` block above?